packages/api-server/src/base/env-config.ts

```diff
@@ -24,6 +24,9 @@ export const envConfig = {
   port: getOptional("PORT"),
   minGasPrice: getOptional("MIN_GAS_PRICE"),
   minSudtFee: getOptional("MIN_SUDT_FEE"),
+  maxQueryNumber: getOptional("MAX_QUERY_NUMBER") || "10000",
```

Comment:
please extract "10000" as `DEFAULT_MAX_QUERY_NUMBER` to packages/api-server/src/db/constant.ts, and keep envConfig optional filed optional with no `||`